internal/beater/server.go

```diff
@@ -222,11 +222,10 @@ func (s server) run(ctx context.Context) error {
 	})
 	g.Go(func() error {
 		<-ctx.Done()
-		// httpServer should stop before grpcServer to avoid a panic caused by placing a new connection into
-		// a closed grpc connection channel during shutdown.
-		// See https://github.com/elastic/gmux/issues/13
-		s.httpServer.stop()
 		s.grpcServer.GracefulStop()
```

Comment:
[to reviewer] Moving GracefulStop first, as this was how the code was originally written before https://github.com/elastic/gmux/issues/13 was discovered. Now that it is fixed, we can revert it, so that it sends GOAWAY to grpc clients.